Sources/System/Errno.swift

```diff
@@ -952,6 +962,7 @@ public struct Errno: RawRepresentable, Error, Hashable, Codable {
   @available(*, unavailable, renamed: "diskQuotaExceeded")
   public static var EDQUOT: Errno { diskQuotaExceeded }
 
+#if !os(WASI)
```

Comment:
`ESTALE` seems available in wasi-libc https://github.com/WebAssembly/wasi-libc/blob/c5264e2bbe532994d06b039005f2af91bedcc1a6/expected/wasm32-wasi/predefined-macros.txt#L299